README.md

```diff
@@ -1,5 +1,13 @@
-This repository contains a list of tests in the Salt Test Suite that are **not** executed. Skipping
-tests is bad and should only be used as a last resort.
+This repository contains a list of tests in the Salt Test Suite that are **not** executed. Remember: **Skipping tests is bad and should only be used as a last resort.**
 
-The list is formatted using the TOML markup language and skipped tests are grouped by file/test level and then by "type". Missing
-groups do  not have t be listed explicitely.
\ No newline at end of file
+The list is formatted using the TOML markup language and skipped tests are grouped by file/test level and then by "type". Missing groups do not have to be listed explicitely. The list is used by `salt-test`, a test launcher that skips the collection of files in the `[ignore]` sections and deselects all tests in the `[skip]` section.
+
+### `ignore` vs `skip`
+
+Rule of thumb: Use `skip` if you can, `ignore` if you must.
+Reason: `ignore` skips over a whole Python module (i.e. a file). It's needed when the import of the module causes errors, e.g. `ImportError`. We don't have all of Salt's Python dependencies packaged and need to exclude files that don't guard their imports.
+`skip` has finer granularity and is prefered for that reason.
+
+### Adding new entries to `skipped_tests.toml`
+
+This repository is "locked" down to ensure all additions to `skipped_tests.toml` are agreed upon by [openSUSE's Salt team](https://github.com/orgs/openSUSE/teams/salt). Therefore, no commits can be pushed to `main` directly. All changes must come via a Pull Request that is approved by at least two other maintainers. All new entries must include a comment that explains why a given test is failing. The Pull Request description should explain why a test case is not getting fixed and instead added to this skiplist. Pull Request reviewers are asked to critically review this explanation.
```

Comment:
[NIT] Simplification. Note that even Github [0] doesn't capitalize pull requests :) 

[0] https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests

```suggestion
This repository is disallows direct commits to `main` to ensure all additions to `skipped_tests.toml` are agreed upon by [openSUSE's Salt team](https://github.com/orgs/openSUSE/teams/salt). All changes must come via a pull request that is approved by at least two other maintainers. All new entries must include a comment that explains why a given test is failing. The pull request description should explain why a test case is not getting fixed and instead added to this skiplist. pull request reviewers are asked to critically review this explanation.
```

Reply:
Wow, I was pretty sure they did. Thanks, I'll change it.

Reply:
Not that I ever used the German docs, but funnily they do use `Pull Request` https://docs.github.com/de/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request. Maybe it was my German brain thinking it should be title-cased :laughing: